.github/ISSUE_TEMPLATE/bug-issues.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Bug
+about: Reporting bugs and problems in Eclipse Codewind
+title: Complete the bug template to report problems found in Eclipse Codewind.
+labels: ''
+assignees: ''
+
+---
+
+**Codewind version:** In which version of Codewind, such as 19.02, did you encounter this issue?
```

Comment:
We are sticking for semantic versioning for now so maybe let's use 0.2.0 as the example?